seed-db.js

```diff
@@ -0,0 +1,190 @@
+  var fragmentCount = Math.random() * 9 + 5;
+  var name = '';
+  for (var i = 0; i < fragmentCount; i++) {
+    name += pickRandom(fragments);
+  }
+
+  name = `${randomLetter().toLocaleUpperCase()}${name.toLocaleLowerCase()}`;
+  for (var i = 0; i < name.length - 1; i++) {
+    if (!digraphs.includes(name.slice(i, i + 2).toLocaleUpperCase())) {
+      name = `${name.slice(0, i + 1)} ${name.slice(i + 1, i + 2).toLocaleUpperCase()}${name.slice(i + 2)}`;
```

Comment:
Is `toLocaleUpperCase()` needed here? It seems to be most useful when used with a locale specified in its parameter. Otherwise, I think `toUpperCase()` should work just fine. Regardless, I don't think there is a performance difference so keeping it is fine too.